src/types/user.ts

```diff
@@ -404,7 +404,7 @@ export default /* GraphQL */ `
     articleCount: Int!
 
     "Number of comments posted by user."
-    commentCount: Int!
+    commentCount: Int! @auth(mode: "${AUTH_MODE.oauth}")
```

Comment:
this field is using by https://github.com/thematters/matters-web/blob/develop/src/components/ArticleDigest/Published/FooterActions/gql.ts#L18